.github/workflows/build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         id: go
 
       - name: Checkout
-        uses: actions/checkout@v1
+        uses: actions/checkout@v3
```

Comment:
With this change, we now need to specify `fetch-depth: 0` so that `goreleaser` has the full history to generate its changelog.